babel.config.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function(api) {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'module' is not defined                    no-undef